benchmarks/gpt_fast/benchmark.py

```diff
@@ -16,14 +21,17 @@ class Experiment:
     metric: str
     target: float
     actual: float
+    model: str
+    quantization: str
+    device: str
 
 
-def do_inference(mod, x, num_samples: int = 5):
+def do_inference(mod, x, num_samples: int = 5, device: str = "cuda"):
```

Comment:
I'm removing this function at #128002 and leverage on ```triton.testing.do_bench```

Reply:
I see, then let me update your other comments and revert this part to avoid conflicts.  I can hold on to this PR and land it after https://github.com/pytorch/pytorch/pull/128002